src/main/java/com/example/paradise/domain/post/api/PostController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.paradise.domain.post.api;
+
+import com.example.paradise.domain.post.application.PostService;
+import com.example.paradise.domain.post.dto.CreatePostRequestDto;
+import com.example.paradise.domain.post.dto.PostResponseDto;
+import com.example.paradise.domain.post.dto.UpdatePostRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/post")
+public class PostController {
+
+    private final PostService postService;
+
+    // 1. 게시글 작성
+    @PostMapping("")
+    public ResponseEntity<PostResponseDto> createPost(@RequestBody CreatePostRequestDto requestDto) {
+        return ResponseEntity.status(HttpStatus.CREATED).body(postService.createPost(requestDto));
```

Comment:
요 `postService.createPost(requestDto)`를 한번 위에서 변수로 담아서 보내주는 건 어떨까 싶어요!!
지금은 내용이 많지 않으니 이미 한눈에 다 보이지만 나중에 디벨롭하다가 코드 수가 늘어나게 되면 이렇게 가독성을 챙겨볼 수 있을 거 같아요.

```suggestion
        PostResponseDto response = postService.createPost(requestDto);
        return ResponseEntity.status(HttpStatus.CREATED).body(response);
```

Reply:
네~ 변수 사용하는 방향으로 수정하겠습니다.